maintainers/maintainer-list.nix

```diff
@@ -948,6 +948,11 @@
     github = "chiiruno";
     name = "Okina Matara";
   };
+  chloekek = {
```

Comment:
Please rewrite your git history into 2 single commits. The first commit should be adding this entry to `maintainer-list.nix` and the second should be the other commits squashed into 1.